package.json

```diff
@@ -68,7 +71,8 @@
     "prettier": "^2.2.1",
     "prettier-plugin-organize-imports": "^1.1.1",
     "prettierx": "^0.17.0",
-    "rimraf": "^3.0.2",
+    "rollup": "^2.38.5",
+    "rollup-plugin-delete": "^2.0.0",
     "typescript": "^4.1.4",
```

Comment:
As you already have modified the package(-lock).json in this PR
Could you update `eslint-plugin-promise` and `typescript` and generate updated package-lock?